clcache.py

```diff
@@ -56,6 +56,7 @@
 #   value: key in the cache, under which the object file is stored
 Manifest = namedtuple('Manifest', ['includeFiles', 'includesContentToObjectMap'])
 
+CompilerArtifacts = namedtuple('CompilerArtifacts', ['objectFilePath', 'compilerStdout', 'compilerStderr'])
```

Comment:
Can the two fields be simplified to?

CompilerArtifacts.stdout
CompilerArtifacts.stderr

I mean, all compiler artifacts come from the compiler.